config/access-om2.yaml

```diff
@@ -315,4 +315,103 @@ sources:
   - metadata_yaml: /g/data/ik11/outputs/access-om2-01/01deg_jra55v150_iaf_cycle1/metadata.yaml
     path:
       - /g/data/ik11/outputs/access-om2-01/01deg_jra55v150_iaf_cycle1
+
+# Added for issue #184
+# Commented out sources have issues to be solved (check comments)
+
+  - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_Control/metadata.yaml
+    path:
+      - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_Control
+
+  # - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthmp/metadata.yaml
+  #   path:
+  #     - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthmp
+  # Same UUID as /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthp/metadata.yaml
+
+  - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_ENFull/metadata.yaml
+    path:
+    - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_ENFull
+
+  # - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthp/metadata.yaml
+  #   path:
+  #   - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthp
+  # Same UUID as /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_qian_wthmp/metadata.yaml
+
+  - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_LNFull/metadata.yaml
+    path:
+    - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_ryf_LNFull
+
+  # - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_ssp245
+  #   path:
+  #   - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_ssp245
+  # Metadata does not exist
+
+  # - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_SAMextr_0203_from9091/metadata.yaml
+  #   path:
+  #     - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_SAMextr_0203_from9091
+  # Metadata fails validation: 
+  # 01 | experiment_uuid : None is not of type 'string'
+  # 02 | realm : None is not of type 'array'
+  # 03 | frequency : 'monthly' is not of type 'array'
+  # 04 | (missing) : 'long_description' is a required property
+
+  # - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_ssp245_mw
+  #   path: 
+  #     - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v13_ryf9091_ssp245_mw
+  # Metadata does not exist
  
+  - metadata_yaml: /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_SAMextr_1011_from9091/metadata.yaml
+    path: 
+      - /g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55_SAMextr_1011_from9091
```

Comment:
This has notes below about metadata validation failing, but it's uncommented. Is that correct?

Reply:
No, it should be commented out (although the recent updates to the build procedure means it will just be ignored, rather than killing the build).